edn/src/edn.rustpeg

```diff
@@ -148,11 +148,11 @@ namespace_separator = "/"
 // TODO: Be more picky here
 // Keywords follow the rules of symbols, except they can (and must) begin with :
 // e.g. :fred or :my/fred. See https://github.com/edn-format/edn#keywords
-symbol_char_initial = [a-z] / [A-Z] / [0-9] / [*!_?$%&=<>]
-symbol_char_subsequent = [a-z] / [A-Z] / [0-9] / [-*!_?$%&=<>]
+symbol_char_initial = [a-z] / [A-Z] / [0-9] / [*!_?$%&=<>.]
```

Comment:
Or we could argue that the Clojure reader has diverged from https://github.com/edn-format/edn